cmd/minishift/cmd/oc_env.go

```diff
@@ -32,31 +32,28 @@ import (
 )
 
 const (
-	ocEnvTmpl = `{{ .Prefix }}PATH{{ .Delimiter }}{{ .OcPath }}{{ .Suffix }}{{ .UsageHint }}`
+	ocEnvTmpl = `{{ .Prefix }}PATH{{ .Delimiter }}{{ .OcDirPath }}{{ .Suffix }}{{ .UsageHint }}`
 )
 
 type OcShellConfig struct {
 	shell.ShellConfig
-	OcPath    string
+	OcDirPath string
 	UsageHint string
 }
 
-func getOcShellConfig(ocPath string, forcedShell string) (*OcShellConfig, error) {
+func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
 	userShell, err := shell.GetShell(forcedShell)
 	if err != nil {
 		return nil, err
 	}
 
 	cmdLine := "minishift oc-env"
 	shellCfg := &OcShellConfig{
```

Comment:
@gbraad , now the error message is
```
> minishift.exe oc-env
Unable find OpenShift client binary.
This might happen when minishift couldn't started properly.
```

We can discuss on error message if it is not appropriate.